Source/Bogus/DataSets/Date.cs

```diff
@@ -25,22 +27,26 @@ public partial class Date : DataSet
       /// <summary>
       /// Create a Date dataset
       /// </summary>
-      public Date(string locale = "en") : base(locale)
+      public Date(string locale = "en", Func<DateTime?> referenceDate = null) : base(locale)
```

Comment:
note: this is a binary breaking change

Reply:
I think I had it that way previously and changed it. Let me revise why. It has been a moment.

Reply:
This does not work like that since both ctors only have optional arguments. I can move the referencedate as a non optional argument to ste start on the second ctor. I dont like that so much but i guess its better.

Reply:
Added the overload